### PR TITLE
PRG/PrivacySettings: 40553, PRG does not have separate 'export_member_data' permission

### PR DESCRIPTION
**Pull request by nhaagen**
https://mantis.ilias.de/view.php?id=40553

### CODE REVIEW

**Review by klees — APPROVED** [review]

**Comment by klees** [comment]
Hi @smeyer-ilias,

would you be so kind and have a look here soon? LGTM, though, but you are listed as implicit maintainer.

Kind regards!

**Comment by smeyer-ilias** [comment]
Thanks, LGTM

**Pull request merged**
